lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -140,14 +140,12 @@ def ask_ftp_file_options
       end
 
       def ask_custom_file_options(server_uri)
-        @filename    = just_ask(*filename_prompt_args) unless action == :restore
-        sample_case  = server_uri.split("/").last
-        hostname     = URI(server_uri).host
-        uri_filename = ask_custom_prompt(hostname, 'filename', "Target filename (e.g.: #{sample_case})")
-        @uri         = server_uri.gsub(sample_case, uri_filename)
+        hostname  = URI(server_uri).host
+        @filename = ask_custom_prompt(hostname, 'filename', "Target filename for backup")
```

Comment:
I realize this was added in the previous PR, and I wasn't really the reviewer, but why are we "passing" the default prompt if this method is only used in one spot?

I was reviewing a similar PR that @carbonin was (where he asked to axe some constants), but this seems like it makes more sense to just have this as a constant, or a frozen string in the method itself instead of passing it from here.

Reply:
Also (not part of this line of code), but I think you can delete the constants at the top of this file for `LOCAL_FILE`, `NFS_FILE`, etc. since we don't use them any more (as of #65).

Separate commit/PR maybe?  I think you would have a lot of tests that would need to be updated, for what it is worth.

Reply:
ok, will make the prompt less generic.

Those constants are used all over the specs :(  They have been simplified quite a bit (now just have `'ftp'` vs the full text string) -- we can hardcode phrases like "ftp" in the specs.